src/com/android/calendar/CalendarController.java

```diff
@@ -71,8 +71,6 @@
     public static final long EXTRA_GOTO_TODAY = 8;
     private static final boolean DEBUG = false;
     private static final String TAG = "CalendarController";
-    private static WeakHashMap<Context, CalendarController> instances =
```

Comment:
Can you change this to leave the original lines in and not the ones in this pull request?